templates/components/streamfield/blocks/card_section_block.html

```diff
@@ -0,0 +1,10 @@
+{% verbatim %}
+{% include "components/streamfield/blocks/heading2_block.html" with large_spacing=True value=value.heading sr_only=value.sr_only_label %}
+
+
+<div class="flex flex-col md:flex-row gap-20">
+    {% for card in value.cards %}
+        {% include "components/streamfield/blocks/card_block.html" with value=card %}
```

Comment:
We can use `{% include_block %}` so we don't have to specify the template again

```suggestion
        {% include_block card %}
```